src/binaryapi/TradingTimes.js

```diff
@@ -119,13 +120,15 @@ class TradingTimes {
                             open: getUTCDate(openTime),
                             close: getUTCDate(close[idx]),
                         }));
+                        symbolOpenTime = (_times[0].open).toLocaleTimeString();
```

Comment:
@nazaninreihani since we're using GTM timezone for all of the projects, here also we should show the time in GTM.